aspnet/SpreadSheet/Editing.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Editing with Spreadsheet widget for Syncfusion Essential JS
-description: Learn about editing support in Syncfusion ASP.NET Web Forms Spreadsheet control, its elements, and more details. 
+title: Editing in the ASP.NET Web Forms Spreadsheet Control | Syncfusion
+description: Learn here about editing support in Syncfusion Essential ASP.NET Webforms Spreadsheet control, its elements and more details. 
```

Comment:
Update as Learn here about editing support in Syncfusion Essential ASP.NET Web Forms Spreadsheet control, its elements, and more details.